gradle-jdks/src/main/java/com/palantir/gradle/jdks/GraalVmCeDistribution.java

```diff
@@ -16,29 +16,21 @@
 
 package com.palantir.gradle.jdks;
 
-import com.google.common.annotations.VisibleForTesting;
 import com.palantir.gradle.jdks.JdkPath.Extension;
 import com.palantir.gradle.jdks.setup.common.Arch;
 import com.palantir.gradle.jdks.setup.common.Os;
-import org.immutables.value.Value;
 
-final class GraalVmCeDistribution implements JdkDistribution {
+public final class GraalVmCeDistribution implements JdkDistribution {
     @Override
     public String defaultBaseUrl() {
         return "https://github.com/graalvm/graalvm-ce-builds/releases/download";
     }
 
     @Override
     public JdkPath path(JdkRelease jdkRelease) {
-        GraalVersionSplit splitVersion = splitVersion(jdkRelease.version());
-
         String filename = String.format(
-                "vm-%s/graalvm-ce-java%s-%s-%s-%s",
-                splitVersion.graalVersion(),
-                splitVersion.javaVersion(),
-                os(jdkRelease.os()),
-                arch(jdkRelease.arch()),
-                splitVersion.graalVersion());
+                "jdk-%s/graalvm-community-jdk-%s_%s-%s_bin",
```

Comment:
It looks like it changed at some point. We probably want to handle downloading old ones too:

* [23.0.2](https://github.com/graalvm/graalvm-ce-builds/releases/tag/jdk-23.0.2): https://github.com/graalvm/graalvm-ce-builds/releases/download/jdk-23.0.2/graalvm-community-jdk-23.0.2_linux-x64_bin.tar.gz
* [22.3.2](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-22.3.2): https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.3.2/graalvm-ce-java17-linux-amd64-22.3.2.tar.gz

I don't exactly know when it changed over.

Reply:
they started to match the graalVM version with the java version, previously for a specific graalvm version (eg. 22.3 - https://www.graalvm.org/release-notes/22_3/ you could install either the java11 or java17 based GraalVM), now when we specify graalVm == 22.3 -> we will be getting the java version == 22.3 https://www.graalvm.org/release-notes/JDK_22/)